### ISSUE TITLE
Do not warn about SQLALCHEMY_DATABASE_URI when SQLALCHEMY_BINDS is set

### ISSUE THREAD

**Issue by krassowski** [mitsuhiko/flask-sqlalchemy#471]
Currently running following test case:
```python
import flask
import flask_sqlalchemy as fsa

app = flask.Flask(__name__)
app.config['SQLALCHEMY_BINDS'] = {
    'foo': 'sqlite://',
    'bar': 'sqlite://'
}
db = fsa.SQLAlchemy(app)

class Foo(db.Model):
    __bind_key__ = 'foo'
    __table_args__ = {"info": {"bind_key": "foo"}}
    id = db.Column(db.Integer, primary_key=True)

class Bar(db.Model):
    __bind_key__ = 'bar'
    id = db.Column(db.Integer, primary_key=True)

class Baz(db.Model):
    id = db.Column(db.Integer, primary_key=True)

db.create_all()
```
Raises:
```
/usr/lib64/python3.5/site-packages/flask_sqlalchemy/__init__.py:819: UserWarning: SQLALCHEMY_DATABASE_URI not set. Defaulting to "sqlite:///:memory:".
  'SQLALCHEMY_DATABASE_URI not set. Defaulting to '
```

I think that it is a good idea to let user know that he or she probably forgotten/misspelled database URI, but in my opinion setting SQLALCHEMY_BINDS should surpass such warning.

**Comment by davidism** [comment]
Fair enough. I hadn't considered that you can use BINDS without a default URL.

**Comment by krassowski** [comment]
I can issue a PR; which branch should I target? (`2.2-maintenance` or `master`?)

**Comment by davidism** [comment]
`2.2-maintenance`, I'll make it part of 2.2.1.

**Comment by davidism** [comment]
closed by #472
